### ISSUE TITLE
#4 memory size of any array

### ISSUE THREAD

**Issue by wangyi041228** [rougier/numpy-100#201]
The question means the total bytes consumed by the element of the array. In order to clear up the misunderstanding, it is suggested to revise the question stem.

Z.nbytes is more direct then Z.size * Z.itemsize.

BTW, 4 typos:
an hint -> a hint
of of -> of
every combinations -> every combination
a two dimensional -> a two-dimensional

References:
https://numpy.org/doc/stable/reference/generated/numpy.ndarray.nbytes.html
https://docs.python.org/3/library/sys.html#sys.getsizeof

**Comment by rougier** [comment]
You're right but the question is a pretext to show size itemize. But we could also illustrate using nbytes == size*itemsize. Do you want to make a PR?
